### PR TITLE
Belong

### PR DESCRIPTION
**Pull request by LizzAlice**
let one device belong to another

### CODE REVIEW

File: devices/forms.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ class DeviceGroupFilterForm(FilterForm):
 
 class DeviceForm(forms.ModelForm):
     error_css_class = 'has-error'
-
+    uses = forms.MultipleChoiceField(choices = Device.objects.none(), required = False)
```
Comment (octomike) [inline devices/forms.py]: Unfortunately, I have a headache with this approach.

In production we have > 4.5k devices with inconsistent names and a multiple choice drop down is pretty much unusable without a proper search. I think we need to reference the associated device by ID here.

File: devices/models.py [hunk 2]
```diff
@@ -128,7 +128,10 @@ class Device(models.Model):
 
     department = models.ForeignKey("users.Department", null=True, blank=True, related_name="devices", on_delete=models.SET_NULL)
     is_private = models.BooleanField(default=False)
+    used_in = models.ForeignKey('self', null = True, blank = True, on_delete = models.SET_NULL,)
 
+    def return_id (self):
+        return self.id
```
Comment (octomike) [inline devices/models.py]: what's this?

File: templates/devices/device_detail.html [hunk 3]
```diff
@@ -568,7 +577,8 @@ <h4>{% trans "Installed software from puppetdb" %}</h4>
         $("#trashbutton").popover({
         html: 'true',
         placement: "left",
-        title : '<span class="text-info" style="margin-right:10px"><strong>{% trans "are you sure?" %}</strong></span>',
+        <!-- title : '<span class="text-info" style="margin-right:10px"><strong>{% trans "Are you sure?" %}</strong></span>', -->
```
Comment (octomike) [inline templates/devices/device_detail.html]: debug foo

File: devices/forms.py (hunk 1 above)
Comment (LizzAlice) [reply devices/forms.py]: Do you mean the inventory number or the serial number? Or the internal id?

File: devices/models.py (hunk 2 above)
Comment (LizzAlice) [reply devices/models.py]: not necessary, now that I think about it XD

File: templates/devices/device_detail.html (hunk 3 above)
Comment (LizzAlice) [reply templates/devices/device_detail.html]: done

File: devices/forms.py (hunk 1 above)
Comment (octomike) [reply devices/forms.py]: Anything that works, really. The device ID is just the first thing that comes to mind.

**Comment by LizzAlice** [comment]
Fixes  #115

**Pull request merged**